src/main/java/seedu/address/logic/parser/ParserUtil.java

```diff
@@ -51,33 +49,6 @@ public static Index parseIndex(String oneBasedIndex) throws ParseException {
         return Index.fromOneBased(Integer.parseInt(trimmedIndex));
     }
 
-    /**
-     * Parses {@code oneBasedIndex} into an {@code Index} and returns it. Leading and trailing whitespaces will be
-     * trimmed.
-     * @throws ParseException if the specified index is invalid (not non-zero unsigned integer).
-     */
-    public static List<Index> parseTwoIndexes(String oneBasedIndexes) throws ParseException {
-        String trimmedIndexes = oneBasedIndexes.trim();
-        String[] splitIndexes = trimmedIndexes.split("\\s+");
-        if (splitIndexes.length != 2) {
-            throw new ParseException(MESSAGE_INVALID_INDEXES);
-        }
-
-        String firstIndex = splitIndexes[0];
-        String secondIndex = splitIndexes[1];
-        if (!StringUtil.isNonZeroUnsignedInteger(firstIndex)) {
-            throw new ParseException(MESSAGE_INVALID_INDEXES);
-        }
-        if (!StringUtil.isNonZeroUnsignedInteger(secondIndex)) {
-            throw new ParseException(MESSAGE_INVALID_INDEXES);
-        }
-
-        List<Index> returnList = new ArrayList<>();
-        returnList.add(Index.fromOneBased(Integer.parseInt(firstIndex)));
-        returnList.add(Index.fromOneBased(Integer.parseInt(secondIndex)));
-        return returnList;
-    }
-
```

Comment:
Wait why are we removing this? Seems important

Reply:
Resolved post discussion:
- Assign and Unassign Command mechanisms were updated and made this function redundant
- No usages of this function was found on IntelliJ